src/main/java/com/conveyal/datatools/manager/jobs/DeployJob.java

```diff
@@ -816,6 +842,10 @@ private String constructUserData(boolean graphAlreadyBuilt) {
         String graphPath = String.join("/", routerDir, OTP_GRAPH_FILENAME);
         //////////////// BEGIN USER DATA
         lines.add("#!/bin/bash");
+        ///// 0. Remove previous status files
+        lines.add("WEB_DIR=/usr/share/nginx/client");
+        lines.add(String.format("rm $WEB_DIR/%s || echo '' > /dev/null", BUNDLE_DOWNLOAD_COMPLETE_FILE));
+        lines.add(String.format("rm $WEB_DIR/%s || echo '' > /dev/null", GRAPH_STATUS_FILE));
```

Comment:
This makes me think: should we also remove previous log and graph build files at this stage?